app/src/main/java/org/mozilla/fenix/settings/address/view/AddressEditorView.kt

```diff
@@ -103,4 +110,20 @@ class AddressEditorView(
             create()
         }.show()
     }
+
+    private fun bindCountryDropdown(state: AddressEditorState) {
```

Comment:
Can you ask in the address team slack if we should just follow Desktop and display the full Country name (eg, United States, Canada)? I doubt we really want to display the key. The designs does indicate "USA", but that could just be a placeholder without further clarification.